### DISCUSSION TITLE
连接断开后怎么确保不使用本地网络

### DISCUSSION THREAD

**Discussion by reheroad** [v2fly/v2ray-core#614] [Q&A]
急

**Comment by solidarity-forever** [comment]
use vm
Create 2 vm, vm A and vm B.

Install proxy soft in vm A, and install browser or other soft which need use the proxy in vm B.

**Change** the type of network card of vm B to "Internal".

**Add** another internal network card to vm A.

Start proxy soft in vm A, and set it's listening ip to 0.0.0.0 or address of internal card in vm A.

Edit proxy config of your soft in vm B, setting the proxy server ip to address of internal card in vm A.

The vm B can not connect to internet directly, it can only connect to other vm in this host whitch is added internal network card.

In this scene, a soft in vm B can only connect to internet by proxy in vm A.

If the connetction of proxy soft in vm A is broken, soft in vm B will not be able to connect to the internet.
